robot.js

```diff
@@ -133,12 +136,10 @@
   }
 
   function turn(turnDirection) {
-    if (turnDirection !== 'left' && turnDirection !== 'right') {
-      console.log('ignoring invalid turn', turnDirection);
-      return;
-    }
-
-    console.log('executing turn()');
+  
+    const invalidTurn = turnDirection !== 'left' && turnDirection !== 'right' ? 'ignoring invalid turn'+ turnDirection : 'executing turn()';
```

Comment:
This looks very much like what I saw earlier in the pull request from dodo. Since you were earlier, I assume you are the originator of this code.